test/com/jambit/DatabaseConnectionTest.java

```diff
@@ -22,49 +21,47 @@
 
   @BeforeAll
   static void init() throws IOException, SQLException {
-    DatabaseConnection.changePropertyPath("config/appTest.properties");
+    DatabaseConnection.setPropertyPath("config/appTest.properties");
     databaseConnection = DatabaseConnection.getInstance();
   }
 
   @Test
   @Order(1)
-  public void insertTest() throws SQLException {
-    MoodEntries testValue =
-        new MoodEntries(0, (int) Math.round(Math.random() * 10), System.currentTimeMillis() / 1000);
-    databaseConnection.writeMoodEntries(testValue);
-    ArrayList<MoodEntries> dbOutputValues = databaseConnection.fetchAllMoodEntries();
-    assertEquals(dbOutputValues.get(dbOutputValues.size() - 1).time, testValue.time);
-    assertEquals(dbOutputValues.get(dbOutputValues.size() - 1).vote, testValue.vote);
+  public void writeIntoDatabase() throws SQLException {
```

Comment:
Did you read the article I've send you?!
In short: 
    * Clear about what each test state is.
    * Specific about the expected behaviour.
template: MethodName_StateUnderTest_ExpectedBehavior
e.g. writeMoodEntry_Persist_ReturnSingleEntry